CovertInfiltration/Src/CovertInfiltration/Classes/UISSManager_CovertAction.uc

```diff
@@ -56,15 +58,20 @@ simulated protected function BuildConfiguration()
 	Configuration = new class'SSAAT_SquadSelectConfiguration';
 	History = `XCOMHISTORY;
 
-	Slots.Length = GetAction().StaffSlots.Length;
+	CovertAction = CovertOpsScreen.GetAction();
```

Comment:
Could you please use `GetAction()` here? It does the same thing but I would prefer to have all code go though one path so that if anything was to change, we would need to change only one place.